spec.html

```diff
@@ -52991,6 +53014,15 @@ <h1>Non-default behaviour in HostMakeJobCallback</h1>
       <h1>Non-default behaviour in HostEnsureCanAddPrivateElement</h1>
       <p>The HostEnsureCanAddPrivateElement abstract operation allows hosts which are web browsers to specify non-default behaviour.</p>
     </emu-annex>
+
+    <emu-annex id="sec-late-errors-for-function-call-assignment-targets">
+      <h1>Late Errors for Function Call Assignment Targets</h1>
+      <p>When a function call (<emu-xref href="#sec-function-calls"></emu-xref>) is used as an assignment target in non-strict code, instead of producing an early error, a *ReferenceError* exception is thrown during evaluation of the assignment.</p>
+      <emu-note>
+        <p>When the assignment target is the |LeftHandSideExpression| of an |AssignmentExpression|, the assignment operator must be `=` or an |AssignmentOperator|; in particular, the allowance here does not apply to the logical assignment operators (`??=`, `&&=`, `||=`).</p>
+      </emu-note>
+      <p>This feature adds step <emu-xref href="#step-assignmenttargettype-web-compat-insertion-point"></emu-xref> to the static semantics of AssignmentTargetType for <emu-grammar>CallExpression : CoverCallExpressionAndAsyncArrowHead</emu-grammar> and <emu-grammar>CallExpression : CallExpression Arguments</emu-grammar>.</p>
```

Comment:
```suggestion
      <p>See step <emu-xref href="#step-assignmenttargettype-web-compat-insertion-point"></emu-xref> of AssignmentTargetType for <emu-grammar>CallExpression : CoverCallExpressionAndAsyncArrowHead</emu-grammar> and <emu-grammar>CallExpression : CallExpression Arguments</emu-grammar>.</p>
```

It's not really a feature, and it's not adding a step.

Reply:
Sounds good to me (I was just sticking hard to the phrasing I saw in #2952).